lib/cisco_node_utils/cmd_ref/acl.yaml

```diff
@@ -0,0 +1,33 @@
+# Command Reference Common ACL
+#
+# For documentation please see:
+#   - README_YAML.md
+#
+---
+_template:
+  config_get: 'show run aclmgr'
+  config_get_token: '/^<afi> access-list <acl_name>$/'
+  config_set: '<afi> access-list <acl_name>'
+
+ace:
+  config_get_token_append: '/^(<seqno>)\s+(\S+)\s+(\S+|\d+)\s+(any|host \S+|\S+\/\d+|\S+ [:\.0-9a-fA-F]+)\s+(eq \S+|neq \S+|lt \S+|gt \S+|range \S+ \S+)*\s*(any|host \S+|\S+\/\d+|\S+ [:\.0-9a-fA-F]+)\s*(eq \S+|neq \S+|lt \S+|gt \S+|range \S+ \S+)*( [a-zA-Z0-9\/ ]*)*$/'
```

Comment:
IMO this regexp is a bit difficult to read and likely will be harder to maintain moving forward.  As mentioned in the previous review I would prefer that we use a less complex regexp in the yaml and handle the regexp parsing in the nodeutils object similar to max_prefix.  Here is a pointer to the max_prefix code which provides more separation of the various elements.

1) NU: https://github.com/cisco/cisco-network-node-utils/blob/develop/lib/cisco_node_utils/bgp_neighbor_af.rb#L352
2) YAML: https://github.com/cisco/cisco-network-node-utils/blob/develop/lib/cisco_node_utils/cmd_ref/bgp_neighbor_af.yaml#L83